installer/dual_rootfs_device.go

```diff
@@ -28,6 +28,14 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	verifyRebootError = "Reboot to new update failed. " +
+		"Expected \"upgrade_available\" flag to be true but it was false. " +
+		"Either the switch to the new partition was unsuccessful, or the bootloader rolled back"
+	verifyRollbackRebootError = "Reboot to old update failed. " +
```

Comment:
suggested rewording: ".. to *the* old update .."

Reply:
Will change!